README.md

```diff
@@ -1,10 +1,38 @@
 # motrpac-tools
 Scripts and Tools useful for MoTrPAC
 
+## make_manifest.py
+
+- Makes manifest for submission MoTrPAC files to  BIC
+- Requires [Python 3.x](https://www.python.org)
+- Examples:
+    - `./make_manifest.py --help`
+
+       _outputs help info_
+
+
+  - `./python make_manifest.py /path/to/upload/ manifest.csv`
+
+     _makes `manifest.cvs` based on files in `/path/to/upload/`_
+
+```
+usage: make_manifest.py [-h] data_path output
+
+Creates manifest for submission to BIC: a comma separated file table of
+relative file paths and md5 sums.
+
+positional arguments:
+  data_path   full base path of the data submission
```

Comment:
Considering that the use case of this script, it might be helpful to expand on what "base path" means in this context. Maybe "Full path to folder containing all files for data submission"?

Reply:
that's more clear. added to py and readme